deployment/ansible/group_vars/all.example

```diff
@@ -9,6 +12,8 @@ production: "{{ 'production' in group_names }}"
 docker_version: "1.10.*"
 docker_py_version: "1.2.3"
 docker_options: "--storage-driver=aufs"
+# meta-variable used to translate app_version into docker container tag
+docker_image_tag: "{% if app_version == 'development' %}latest{% else %}{{app_version}}{% endif %}"
```

Comment:
Is it expected that `app_version` will always be `development` when you're developing locally?

Reply:
No; `app_version` is ignored completely if developing locally. I'll make that clearer in the comment up top.